### PR TITLE
Prepare driver for v2.0.0 release

### PR DESCRIPTION
**Pull request by justing-bq**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by kylepbit — COMMENTED** [review]
+1

File: go.sum [hunk 1]
```diff
@@ -1,15 +1,14 @@
-github.com/amzn/ion-go v1.1.0 h1:699A2uZDtkPf5yTMcRkTMoCBYa0UELLgAt0k8Psw+RY=
-github.com/amzn/ion-go v1.1.0/go.mod h1:93Bu1K0O/CDosTCDzJ1cNY7XpdHGAGE2NepHcGsiV70=
-github.com/amzn/ion-hash-go v1.1.0 h1:m7UeSm77kENRJCI1IbcRVRZNwh4/Et2uQoFPf+1x5e0=
-github.com/amzn/ion-hash-go v1.1.0/go.mod h1:UF0NLRWtcpfgBneeqK0ESnN7iSmqpRvuY9VHvD4XwNg=
+github.com/amzn/ion-go v1.1.2/go.mod h1:7wQBWQ7PhPpZCr9PL+mtuIyNmyLjuV8qt2mrfxmvkA8=
```
Comment (byronlin13) [inline go.sum:1]: could you run `go mod tidy` ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,18 @@
-# 1.1.0
+# 2.0.0
+
+* Bumped Ion Go to `v1.1.3` allow support for unmarshalling Ion timestamps to Go time objects.
+* Logging interface now provides the LogLevel as a parameter.
+* Expose transaction ID in Transaction interface.
+
+# 1.1.1
```
Comment (byronlin13) [inline CHANGELOG.md:7]: Could we add the date beside the version? (like we do for 1.1.0) 

The released date is: `2021-06-16)`

Comment (justing-bq) [reply CHANGELOG.md]: Done.

File: go.sum (hunk 1 above)
Comment (justing-bq) [reply go.sum:1]: I ran the command but it made no difference.

**Review by byronlin13 — APPROVED** [review]

**Pull request merged**